todo-list-1/src/Components/AddTask/AddTask.js

```diff
@@ -0,0 +1,56 @@
+import React, { useState } from "react";
+
+import "./AddTask.css";
+import plus from "../../Images/plus.png";
+
+function AddTask({ onAddTodo }) {
+  const [enterTodo, setEnteredTodo] = useState([]);
+  const [clicked, setClicked] = useState(false);
+  function clickHandler() {
+    !clicked ? setClicked(true) : setClicked(false);
+  }
+  // on form submission handler
+  function addNewTaskHandler(event) {
+    event.preventDefault();
+    // handling  not giving input
+    if (enterTodo === "") {
+      alert("Enter the todo.");
+    } else {
+      const defaultTodos = {
+        id: Math.random().toString(),
+        title: enterTodo,
+      };
+      onAddTodo(defaultTodos);
+      setEnteredTodo("");
+    }
+  }
+  const Escape = (e) => {
+    if (e.code === "Escape") {
+      setClicked(false);
+    }
+  };
+  return (
+    <div className="add-task-container">
+      <form
+        className={clicked ? "form-container" : "form-hidden"}
```

Comment:
instead of using class-name for hiding form, you should use conditional rendering